### ISSUE TITLE
Enable language localization for card effects

### ISSUE THREAD

**Issue by Phrancis** [Cardshifter/Cardshifter#278]
If we extracted the code that translates the Groovy DSL from cards to plain English, we should be able to have a language selector in the GUI to enable other languages. There are a few other things that would need to be modified as well for the rest of the GUI, perhaps extracting the different buttons and game messages to a data file, like XML or JSON. I'm thinking that if the Rubberduck guys can do it, so can we!

**Comment by Zomis** [comment]
Sure we can, but is it something we really should give priority to? I think in gaming, it is far more common to have everything in English.

**Comment by Phrancis** [comment]
I don't think we should _prioritize_ it per se, but I think we should do it at some point in the future.

**Comment by jacwah** [comment]
> If we extracted the code that translates the Groovy DSL from cards to plain English

I'm currently working on this to make changing the descriptions easier. Translating that part would then be trivial.

There are lots of other things too though, card names and flavor come to mind.
